.travis-ci.sh

```diff
@@ -75,6 +75,16 @@ function build_one {
     echo "End   availability check."
     case $TRAVIS_OS_NAME in
     linux)
+      # we need fresh gcc and binutils, maybe...
+      if [ `opam install --dry-run $pkg | grep -c mirage-entropy-xen` -gt 0 ] ; then
```

Comment:
likely, but I'm not an expert in launchpad/ppa (never used it)... if you can easily fork + build `gcc >=4.8` and `binutils >=2.24`, that'd be great... furthermore, I've no clue how opam depext tags are defined and handled...